src/FRP/Event.purs

```diff
@@ -46,6 +50,18 @@ foreign import never :: forall a. Event a
 instance functorEvent :: Functor Event where
   map = mapImpl
 
+instance filterableEvent :: Filterable Event where
+  filter = filter
```

Comment:
Not sure I understand fully, sorry >_< I changed the FFI import to `filterImpl`, and updated the references in the `Filterable`. Should I also add a `filter = filterImpl` outside the `Filterable` class and re-export?